### PR TITLE
Update OCBC Digital logo

### PR DESCRIPTION
**Pull request by vault087**

### CODE REVIEW

**Review by danfowler — APPROVED** [review]

**Comment by danfowler** [comment]
@vault087 I think we need to update the sonarcloud key to make the pipeline pass

**Comment by danfowler** [comment]
@vault087 I've updated and the pipeline passes

**Comment by danfowler** [comment]
@vault087 This is different from how we implemented on Android https://github.com/omise/omise-android/pull/285

File: OmiseSDK/Client.swift [hunk 1]
```diff
@@ -331,7 +331,7 @@ extension Client {
     // swiftlint:disable:next function_body_length
     private static func completeRequest<T: Object>(_ request: Request<T>, callback: Request<T>.Callback?) -> ((Data?, URLResponse?, Error?) -> Void) {
         // swiftlint:disable:next closure_body_length
-        return { (data: Data?, response: URLResponse?, error: Error?) -> Void in
```
Comment (aashishgurung) [inline OmiseSDK/Client.swift:334]: Curious why we are removing it?

**Review by danfowler — APPROVED** [review]

**Review by aashishgurung — APPROVED** [review]

**Pull request merged**